themes/default/content/docs/intro/console/webhooks.md

```diff
@@ -33,27 +33,31 @@ will be notified whenever a stack is updated or changed. Organization
 webhooks will be notified for events happening within each of the organization's
 stacks.
 
-> There are some restrictions for the number of webhooks that can be registered
-> when using the Pulumi Team editions. [Contact us]({{< relref "/about#contact-us" >}})
-> if you need the limit increased.
+{{% notes "info" %}}
+There are some restrictions for the number of webhooks that can be registered
+when using the Pulumi Team editions. [Contact us]({{< relref "/about#contact-us" >}})
+if you need the limit increased.
+{{% /notes %}}
```

Comment:
is this still accurate?  I haven't seen this configurable per org.

Reply:
not sure, do you know who is the right person to ask?

Reply:
It is Chris, but I think leaving as is for now works since he is on PTO.